components/busola-migrator/internal/config/config.go

```diff
@@ -19,6 +19,7 @@ type Config struct {
 	TimeoutIdle    time.Duration `envconfig:"default=120s"`
 	BusolaURL      string        `envconfig:"default=https://busola.main.hasselhoff.shoot.canary.k8s-hana.ondemand.com"`
 	StaticFilesDIR string        `envconfig:"optional"`
+	KubeconfigID   string        `envconfig:""`
 
 	OIDC OIDCConfig
```

Comment:
If we no longer use oidc related values in the business logic I'd remove them from configs.